presto-docs/src/main/sphinx/release/release-0.147.rst

```diff
@@ -11,6 +11,7 @@ General Changes
 Hive Changes
 ------------
 
+* Fix reading symlink file format when the symlink crosses filesystem boundaries.
```

Comment:
The original wording is more accurate. It could be an HDFS file pointing to S3, for example.